web/concrete/authentication/concrete/controller.php

```diff
@@ -1,7 +1,8 @@
 <?php
-namespace Concrete\Authentication\Concrete;
+namespace concrete\authentication\concrete;
```

Comment:
@mlocati @KorvinSzanto 

Thanks for your feedback guys, it makes perfect sense. I've already pushed the correct namespace. Thanks!